### DISCUSSION TITLE
[Feature] Hide albums from History panel

### DISCUSSION THREAD

**Discussion by LucaTheHacker** [immich-app/immich#2838] [Feature Request]
### The feature

Currently Immich only allows to create albums and still show all the pictures in the "Photos" tab.
This is fine if you want to just look at your pictures in historical order, but is a mess when you like organization, or if you're trying to create new Albums.

In my opinion it would be nice to have a view that does not include pictures that are already in albums, as "I already know where to find them".

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Related to #1645
